kinto/plugins/accounts/utils.py

```diff
@@ -1,9 +1,15 @@
 import bcrypt
 
 
+# XXX: THIS IS WRONG OF COURSE. But it is the only way I found to have the password
```

Comment:
:joy:

Reply:
But then I don't have it in the `on_account_changed` event to delete the cache entries.

In that case we could introduce a new feature in the cache backend to delete multiple keys with `*` (`account:alice:*:verified`)